<launch>
	<!-- run the position_control.cpp -->

	<node pkg="px4_command" type="px4_pos_controller" name="px4_pos_controller" output="screen">

		<!-- load blacklist, config -->
		<rosparam command="load" file="/home/nvidia/px4_ws/src/px4_command/launch/PID_param.yaml" />


	</node>
</launch>
